### PR TITLE
routesrv: reuse skipper.Options 

### PR DESCRIPTION
**Pull request by AlexanderYastrebov**
Reuse skipper.Options datastructure to avoid duplicate work of adding
the same flags/options to both skipper and routesrv.

Followup on https://github.com/zalando/skipper/pull/1873

- [x] requires https://github.com/zalando/skipper/pull/2419

Signed-off-by: Alexander Yastrebov <alexander.yastrebov@zalando.de>

### CODE REVIEW

File: remote_redis_test.go [hunk 1]
```diff
@@ -0,0 +1,554 @@
...
+	"github.com/zalando/skipper/routesrv"
+	"github.com/zalando/skipper/routing"
+	"github.com/zalando/skipper/routing/testdataclient"
+	"github.com/zalando/skipper/scheduler"
+	"github.com/zalando/skipper/tracing/tracingtest"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestConcurrentKubernetesClusterStateAccessWithRemoteRedis(t *testing.T) {
```
Comment (MustafaSaber) [inline remote_redis_test.go:35]: I think you only need this test case and others could be left in skipper package

Comment (AlexanderYastrebov) [reply remote_redis_test.go]: Fixed by https://github.com/zalando/skipper/pull/2419

**Comment by MustafaSaber** [comment]
👍

File: skipper.go [hunk 2]
```diff
@@ -891,6 +891,39 @@ type Options struct {
 	LuaSources []string
 }
 
+func (o *Options) KubernetesDataClientOptions() kubernetes.Options {
```
Comment (MustafaSaber) [inline skipper.go:894]: Just one question here, is it better to have this as Options function or to send options reference?

**Comment by RomanZavodskikh** [comment]
So, Skipper and Routesrv options set are completely identical, aren't they?

**Comment by RomanZavodskikh** [comment]
:+1:

File: config/config.go [hunk 3]
```diff
@@ -21,7 +21,6 @@ import (
 	"github.com/zalando/skipper/eskip"
 	"github.com/zalando/skipper/net"
 	"github.com/zalando/skipper/proxy"
-	routesrv "github.com/zalando/skipper/routesrv"
```
Comment (szuecs) [inline config/config.go:24]: :tada:

**Comment by AlexanderYastrebov** [comment]
> So, Skipper and Routesrv options set are completely identical, aren't they?

yes

**Pull request merged**